include/ck_tile/host/reference/reference_gemm.hpp

```diff
@@ -19,12 +20,16 @@ template <typename ADataType,
 CK_TILE_HOST void reference_gemm(const HostTensor<ADataType>& a_m_k,
```

Comment:
We could have layouts for each tensor as tparams here. Take a look at old ck reference gemm.